std/traits.d

```diff
@@ -6524,6 +6575,41 @@ unittest
         static assert(mostNegative!T == 0);
 }
 
+/**
+Get the type that a scalar type `T` will $(LINK2 https://dlang.org/spec/type.html#integer-promotions, promote)
```

Comment:
> That's been settled in the favor of HTTP or HTTPS.

No, it hasn't. Everyone agrees that WEB can go away, but I don't agree that HTTP/HTTPS are to be preferred over LINK2. See [my reply](https://github.com/dlang/phobos/pull/4411#issuecomment-226392170) in the linked PR.